test/runnable/test14874.d

```diff
@@ -0,0 +1,36 @@
+template indexOfReturn(T...)
+{
+    static if (T.length == 0)
+    {
+        return -1;

```

Comment:
And, you can add the test code in the test file `runnable\traits.d`. Correcting `__traits()` related test cases into that source file would be better.